### DISCUSSION TITLE
Who is using Docusaurus v1

### DISCUSSION THREAD

**Discussion by slorber** [facebook/docusaurus#6655] [v1-v2 migration]
We are trying to automate the v1 to v2 migration with a CLI.

If you own a Docusaurus v1 open-source website, please report it here. 
We want to see if the migration cli works on your site.

We already know some of those sites, from the spreadsheet of #1834, so it's not necessary to report if you are already in that list:

https://docs.google.com/spreadsheets/d/18SWTklRJVvOTOQEStTo8LXBc78ZMFdbdwp2EEJK5q8g/edit?usp=sharing

Thanks

**Comment by shirakaba** [comment]
React NativeScript’s docs are on v1: https://github.com/shirakaba/react-nativescript-site

Site hosted with Netlify here: https://react-nativescript.netlify.app/

**Comment by JapneetSingh5** [comment]
https://webdriver.io/ is on v1
Here is the repository https://github.com/webdriverio/webdriverio

**Comment by SwaroopH** [comment]
[EthVigil](https://EthVigil.com) is on v1. 

_EDIT: We may be merging this soon, added a branch of v1_

Repo: https://github.com/blockvigil/ethvigil-docs/tree/docusaurus-v1

PR tracking v2 migration: https://github.com/blockvigil/ethvigil-docs/pull/22

**Comment by mkolar** [comment]
Hey guys.

We still run docusaurus v1 at https://pype.club

I've played a bit with a migration to v2 a few months back, but had to drop it for other things. Would love to migrate in near future.

**Comment by slorber** [comment]
Hey,

The migration cli has been released, if you want, try it, and we can help in Discord

**Comment by slorber** [comment]
Many projects of the scala ecosystem are on v1 and use a solution called mdoc: https://scalameta.org/mdoc/docs/docusaurus.html#installation
